cms/djangoapps/contentstore/management/commands/add_test_transcript.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think we can remove this file now. you can keep it on your machine for future purpose if needed.